app/controllers/order_item_controller.rb

```diff
@@ -0,0 +1,7 @@
+class OrderItemController < ApplicationController
+  def index
+    order_id = params[:order_id]
+    render "index", locals: {order_id: order_id}
```

Comment:
This is one approach of passing of variables to view.
You can also set it to instance variable and get rid of render line.
```suggestion
    @order_id =  params[:order_id]
```